sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_workspace_hub/workspace_hub.py

```diff
@@ -40,3 +40,4 @@ class WorkspaceHubSchema(PathAwareSchema):
     managed_network = ExperimentalField(NestedField(ManagedNetworkSchema))
     existing_workspaces = fields.List(fields.Str())
     workspace_hub_config = ExperimentalField(NestedField(WorkspaceHubConfigSchema))
+    enable_data_isolation = fields.Bool()
```

Comment:
Please modify the appropriate schema unit test to assign and verify this value at least once

Reply:
Updated